pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -287,6 +289,11 @@ def pre_configure_ddp(self):
             )
             self._ddp_kwargs["find_unused_parameters"] = True
 
+    def _wrap_model(self) -> None:
+        self._model = DistributedDataParallel(
+            LightningDistributedModule(self.model), device_ids=self.determine_ddp_device_ids(), **self._ddp_kwargs
+        )
+
```

Comment:
since wrap model is a 1-line change used only in one spot, we don't need a separate function for it here. we could inline this directly in `configure_ddp`